create-vm.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -euo pipefail
+
```

Comment:
Could you include a link back to http://jlebon.com/devconf/slides.pdf here?
I would appreciate the attribution. :)